### ISSUE TITLE
Particle effect options not working

### ISSUE THREAD

**Issue by Sago92** [SpongePowered/Sponge#2515]
Dear Sponge developers,

I am currently working with particle effects. I want to show in a area of 10 blocks a green dust effect. This effect should show the heal range of an ability that heals all players inside this range.

To realize that I used the following code:

```
ParticleEffect particleEffect = ParticleEffect.builder()
   .type(ParticleTypes.REDSTONE_DUST)
   .quantity(250)
   .option(ParticleOptions.COLOR, Color.GREEN)
   .option(ParticleOptions.SCALE, 2D)
   .build();
seedLocation.getExtent().spawnParticles(particleEffect, seedLocation.getPosition(), 10);
```

The result I get looks like this:

![Unbenannt](https://user-images.githubusercontent.com/32706518/75155240-e2d5a680-570f-11ea-9500-dff7af0f8629.png)

It seems the options did not apply, even if I change the particle type. I am using the latest API and Sponge Forge versions.

**Sponge Forge:** spongeforge-1.12.2-2838-7.1.10-RC3998
**Sponge API:** org.spongepowered:spongeapi:7.2.0-20200130.194701-33

I would appreciate any support.

Regards 
Sascha

**Comment by dualspiral** [comment]
The colour one does need checking, but it is well worth noting that not all options work with all particle effects. For example, [`SCALE` doesn't work with redstone dust, we specify as much](https://jd.spongepowered.org/7.1.0/org/spongepowered/api/effect/particle/ParticleOptions.html#SCALE).

Remove invalid options and try again, just in case there is some weird case where an invalid option causes things to do this.

**Comment by Sago92** [comment]
Didn't know that the available options where mentioned in the java doc. Thanks for the hint. Next time I will take a look into the java doc aswell.

Nevertheless I tested to remove the scale option, but the redstone color is still red and the range is still 1 block instead of 10.

And one more thing. The [quantity](https://jd.spongepowered.org/7.1.0/org/spongepowered/api/effect/particle/ParticleOptions.html#QUANTITY) option is working for redstone dust, but not listed inside the java doc. And to be sure it won't prevent the color from changing I tested it only with the color option aswell.

**Comment by Cybermaxke** [comment]
You're looking for the `OFFSET` option, this will spread particles in a square area around the position. The range provided is only to make the effect visible to players within the range around the particle. The scale also only works for a few types.

**Comment by Sago92** [comment]
I learned a lot today. Thanks for the informations.
Now only the not working color change is open.

**Comment by Sago92** [comment]
Any progress here?

**Comment by Cybermaxke** [comment]
Can you try to use orange as a color?

**Comment by Sago92** [comment]
There is no predefined orange color available 

![Unbenannt](https://user-images.githubusercontent.com/32706518/75889232-e3b3ba00-5e2c-11ea-877e-424d8d4240b5.png)

I tried it with the ofRgb method but the result is the same.

`.option(ParticleOptions.COLOR, Color.ofRgb(255, 128, 0))`

The particle color is still red even if I change it to blue or yellow. I also tried this on the newest Sponge Forge version.
